hasura.planx.uk/migrations/1669979245259_alter_table_public_analytics_logs_alter_column_analytics_id/up.sql

```diff
@@ -0,0 +1,4 @@
+-- There was an inconsistency with `analytics` and `analytics_logs`.
+-- The `analytics_id ` column in `analytics_logs` table had an `int` value, but the `analytics`'s id is a `bigint`.
+-- This causes a few issues when we are querying on GraphQL using variables.
+ALTER TABLE "public"."analytics_logs" ALTER COLUMN "analytics_id" TYPE int8;
```

Comment:
should this be `TYPE bigint` rather than `int8` ? 

via https://www.postgresql.org/docs/9.1/datatype-numeric.html
> SQL only specifies the integer types integer (or int), smallint, and bigint. The type names int2, int4, and int8 are extensions, which are also used by some other SQL database systems.

also minor nit: i personally appreciate bug fixes as separate PRs if they are completely un-related to existing open PRs !